packages/invalid-dependency/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export function invalidFunction(message: string) {
+export const invalidFunction = (message: string) => () => {
```

Comment:
Should the function return a function here?
```suggestion
export const invalidFunction = (message: string) => {
```